server/TEST.md

```diff
@@ -1,157 +0,0 @@
-An h1 header
```

Comment:
As I remember, @AleshaOleg created this file for test purpose while he worked on the rendering of markdown.
Now, rendering is working, so I thought, that using of README.MD from out project instead TEST.MD is more adequate.
But if you think that we still need this file, I can create one and make push :)